tools/.style.yapf

```diff
@@ -0,0 +1,3 @@
+[style]
```

Comment:
any particular reason to hide this as a dot-file, as opposed to just calling this style.yapf?

Reply:
This can be called anything since we are calling yapf programmatically but using this name ensures that the file is discovered automatically by [yapf](https://github.com/google/yapf/#formatting-style) if run from the command line.

```md
YAPF will search for the formatting style in the following manner:

1. ...
2. In the [style] section of a .style.yapf file in either the current directory or one of its parent directories.
3. ...
```

Reply:
I'm not sure why we'd want to run it from the command-line...we have a script wrapper now, right?

Reply:
We do and I'm open to changing it. I just kept this for the off-chance someone tried to.

Another place yapf will check is a `setup.cfg` file which is also a standard place for formatting rules. This would also mean the file isn't hidden. What do you think about using that instead?